components/ruby/lib/chef-licensing/license_key_fetcher/file.rb

```diff
@@ -131,6 +131,8 @@ def fetch_or_persist_url(license_server_url_from_config, license_server_url_from
           load_basic_license_data_to_contents(url, [])
         elsif @contents && license_server_url_from_system && license_server_url_from_system != @contents[:license_server_url]
           @contents[:license_server_url] = license_server_url_from_system
+        elsif @contents && ChefLicensing::Config.license_server_url_in_config_file && (license_server_url_from_config && license_server_url_from_config != @contents[:license_server_url])
```

Comment:
Is this intended that the license server url from system has higher priority than the license server url set in the config block?